### PR TITLE
Don't import recursively

### PR DESCRIPTION
**Pull request by mjoyce91**
Fixes an issue when trying to migrate

### CODE REVIEW

File: talentmap_api/fsbid/services/employee.py [hunk 1]
```diff
@@ -43,22 +43,6 @@ def get_employee_information(jwt_token, emp_id):
     except:
         return {}
 
-def get_user_information(jwt_token, perdet_seq_num):
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/employee.py:46]: 😭

**Comment by elizabeth-jimenez** [comment]
will putting `from talentmap_api.fsbid.services.client import get_user_information` in talentmap_api/user_profile/serializers.py also not be a recursive import?

**Comment by mjoyce91** [comment]
> will putting from talentmap_api.fsbid.services.client import get_user_information in talentmap_api/user_profile/serializers.py also not be a recursive import?

@elizabeth-jimenez It shouldn't be, but I ran into a similar issue even when I imported the specific function and not the entire file

**Comment by elizabeth-jimenez** [comment]
> > will putting from talentmap_api.fsbid.services.client import get_user_information in talentmap_api/user_profile/serializers.py also not be a recursive import?
> 
> @elizabeth-jimenez It shouldn't be, but I ran into a similar issue even when I imported the specific function and not the entire file

@mjoyce91 hehe i think i wasn't being clear. I figured we needed that new import, since get_user_information had now been moved to another file. So my question was more asking if it was intentional that it was left out, but I see it was added and it all works now!

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🍭 😭 🍬 🥮  👍

**Pull request merged**